pkg/logql/downstream.go

```diff
@@ -379,7 +387,8 @@ func (ev *DownstreamEvaluator) NewStepEvaluator(
 			}
 		}
 
-		results, err := ev.Downstream(ctx, queries)
+		acc := newQuantileSketchAccumulator()
+		results, err := ev.Downstream(ctx, queries, acc)
```

Comment:
This is a little annoying since we could call `acc.Result`. I didn't want to chan the logic too much.